src/containers/navbar/navbar.component.jsx

```diff
@@ -1,14 +1,15 @@
 import React from "react";
 import { Route, Link, BrowserRouter as Router } from "react-router-dom";
-import { useSelector } from "react-redux";
 
 const Navbar = (props) => {
   return (
     <div className="border-grayy align-middle border-b-2 flex justify-between pt-3 pb-1 text-l bg-bluey">
-      <div>
+      <Link to="/side-bar">
```

Comment:
Does this need to be a `Link`? A `Link` only serves you change the url right? This is just a side-bar?

Reply:
Better as a modal?

Reply:
Just a div will go. You can google around to see simple side-bar implmentations
Here is an example:
https://medium.com/javascript-in-plain-english/create-a-reusable-sidebar-component-with-react-d75cf48a053a